### DISCUSSION TITLE
Latex plugin

### DISCUSSION THREAD

**Discussion by pierrethiriet** [quasarframework/quasar-ui-qmarkdown#370] [General]
Hi, and thanks for the __QMarkdown__ Plugin.
I do have a minor issue using latex plugins with QMarkdown v2. 
More specifically, I'll try using [markdown-it-texmath](https://github.com/goessner/markdown-it-texmath) in the following way:
```js
<template>
  <q-page padding>
    <!-- eslint-disable vue/html-indent -->
    <q-markdown  :plugins="plugins"  >
  if you solve $1+2$ you get $3
  $a$
  $$\varphi$$
    </q-markdown>
  </q-page>
</template>

<script setup>
import { QMarkdown } from '@quasar/quasar-ui-qmarkdown'
import '@quasar/quasar-ui-qmarkdown/dist/index.css'
import markdownItTexMath from 'markdown-it-texmath/texmath'
import 'markdown-it-texmath/css/texmath.css'

const plugins = [markdownItTexMath]
</script>

```

And all the latex expressions are double, one with a proper style and a second unstyled. The same happens with another latex parser (markdown-it-katex).

![Capture d’écran 2022-05-24 150228](https://user-images.githubusercontent.com/33446598/170041909-9643c7ec-d132-40d1-af6e-b409dd65a05a.png)

Is it related to the latex plugins? Or in the way of declaring the plugin? Or is there some limitation with plugin compatibility of QMarkdown?
Thanks.
Pierre

**Comment by hawkeye64** [comment]
I'll need to investigate this further to see what the issue might be.

**Reply by pierrethiriet** [reply]
Ok. Thanks !

**Reply by hawkeye64** [reply]
My apologies as I have not had time to look into this. I am donating my previous laptop, so the dev environment is gone and I have not had time to set it up on the new laptop yet. As soon as I am able, I will look into this.

**Reply by pierrethiriet** [reply]
No problem !

**Comment by hawkeye64** [comment]
I have added an example to the docs for the upcoming release of v3

<img width="1156" height="392" alt="image" src="https://github.com/user-attachments/assets/db44e9be-127b-48c2-a768-62f17138ff38" />
